compareECLFiles.cmake

```diff
@@ -1,3 +1,13 @@
+# This script manages the addition of tests.
+# The tests are orchestrated by a shell script,
+# configured using opm_set_test_driver()
+# and then the appropriate helper macro is called to
+# register the ctest entry through the opm_add_test macro.
+# Information such as the binary to call and test tolerances
+# are passed from the build system to the driver script through
+# command line parameters. See the opm_add_test() documentation for
+# details on the parameters passed to the macro.
+
 # Set absolute tolerance to be used for testing
 set(abs_tol 2e-2)
```

Comment:
Question: Are the tolerances `abs_tol` and `rel_tol` properly scoped within one macro definition, or will settings applied within one macro leak over the following macro (invocations).